flytekit/core/array_node_map_task.py

```diff
@@ -347,6 +349,43 @@ def _raw_execute(self, **kwargs) -> Any:
 
 
 def map_task(
+    target: Union[LaunchPlan, PythonFunctionTask],
+    concurrency: Optional[int] = None,
+    min_successes: Optional[int] = None,
+    min_success_ratio: float = 1.0,
+    **kwargs,
+):
+    """
+    Wrapper that creates a map task utilizing either the existing array_node_map_task
+    or the drop in replacement ArrayNode implementation
+    Args:
+        target: The argument is a Flyte entity of which will be mapped over
+        concurrency: If specified, this limits the number of mapped tasks than can run in parallel to the given batch
+        size. If the size of the input exceeds the concurrency value, then multiple batches will be run serially until
+        all inputs are processed. If set to 0, this means unbounded concurrency. If left unspecified, this means the
+        array node will inherit parallelism from the workflow
+        min_successes: The minimum number of successful executions
+        min_success_ratio: The minimum ratio of successful executions
+        successfully before terminating this task and marking it successful.
+    """
```

Comment:
Reformat this docstring to use the [reST format](http://daouzli.com/blog/docstring.html#id26) (e.g. [[1]](https://github.com/flyteorg/flytekit/blob/aa0b7c70450a5ff65956e4964f48f33649a25233/flytekit/configuration/file.py#L298-L304) or [[2]](https://github.com/flyteorg/flytekit/blob/aa0b7c70450a5ff65956e4964f48f33649a25233/flytekit/core/array_node_map_task.py#L44-L51)).